mzLib/Omics/Digestion/LysisProduct.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Omics.Modifications;
+
+namespace Omics.Digestion
+{
+    public class LysisProduct
```

Comment:
Is this supposed to be an abstract?

Reply:
Made this change and adjusted its constructor to be protected